eventstream_rpc/source/EventStreamClient.cpp

```diff
@@ -213,9 +213,9 @@ namespace Aws
             }
         }
 
-        bool ConnectionLifecycleHandler::OnErrorCallback(int errorCode)
+        bool ConnectionLifecycleHandler::OnErrorCallback(RpcError error)
         {
-            (void)errorCode;
+            (void)error;
             /* Returning true implies that the connection will close upon receiving an error. */
```

Comment:
Missed this before, but the documentation should read "the connection _should_ close as a result of encountering this error"